src/central-server/admin-service/ui/src/views/GlobalConfiguration/shared/ConfigurationAnchor.vue

```diff
@@ -129,6 +140,24 @@ export default Vue.extend({
         .fetchConfigurationAnchor(this.configurationType)
         .finally(() => (this.loading = false));
     },
+    downloadConfigurationAnchor() {
+      this.loading = true;
+      this.configurationSourceStore
+        .downloadConfigurationAnchor(this.configurationType)
+        .finally(() => (this.loading = false));
+    },
+    recreateConfigurationAnchor() {
+      this.loading = true;
+      this.configurationSourceStore
+        .recreateConfigurationAnchor(this.configurationType)
+        .finally(() => (this.loading = false));
+    },
+    showDownloadAnchorButton(): boolean {
+      return this.hasPermission(Permissions.DOWNLOAD_SOURCE_ANCHOR);
+    },
+    showRecreateAnchorButton(): boolean {
```

Comment:
showDownloadAnchorButton and showRecreateAnchorButton can be added as computed properties/methods